closed/test/jtreg-ext/requires/OpenJ9PropsExt.java

```diff
@@ -54,7 +55,8 @@ public Map<String, String> call() {
             map.put("vm.musl", "false");
             map.put("vm.openj9", "true");
             map.put("vm.opt.final.ClassUnloading", "false");
-        }
+            map.put("vm.opt.final.ZGenerational", "false");
+       }
```

Comment:
Please undo the change to this line, unless you'd like to join the `catch` clause to it (but the indentation was correct).

Reply:
Updated